src/shared/locales/en/translation.json

```diff
@@ -314,7 +314,8 @@
         "tag": "Tag",
         "generateAnAddress": "Generate address",
         "generateAnAddressTitle": "Generate an address",
-        "generateAnAddressExplanation": "Please generate an address before using this function."
+        "generateAnAddressExplanation": "Please generate an address before using this function.",
+        "generateAddressInformation": "Generate a new deposit address every time you wish to receive IOTA"
```

Comment:
If the following situation happens, users will be confused by this message.

1. User wants to withdraw from an exchange
2. User generates address A
3. User requests the withdrawal
4. User wants to receive a transaction from a friend
5. User generates an address, but address A shows again (the exchange withdrawal has not been sent)
6. User is confused because the message told it to generate a **new** deposit address but it still shows address A